tests/testsuite/update.rs

```diff
@@ -1646,21 +1646,124 @@ fn update_breaking_unstable() {
         .file(
             "Cargo.toml",
             r#"
+            [package]
+            name  =  "foo"
+            version  =  "0.0.1"
+            edition  =  "2015"
+            authors  =  []
+        "#,
+        )
+        .file("src/lib.rs", "")
+        .build();
+
+    p.cargo("update --breaking")
+        .masquerade_as_nightly_cargo(&["update-breaking"])
+        .with_status(101)
+        .with_stderr_unordered(
+            "\
+[ERROR] the `--breaking` flag is unstable, pass `-Z unstable-options` to enable it
+See https://github.com/rust-lang/cargo/issues/12425 for more information about the `--breaking` flag.
+",
+        )
+        .run();
+}
+
+#[cargo_test]
+fn update_breaking_dry_run() {
```

Comment:
Could you have this separate out in the prior commit?

Reply:
Fixed.